cfme/infrastructure/provider/virtualcenter.py

```diff
@@ -102,10 +102,10 @@ def get_console_connection_status(self):
         return self._try_element_lookup(self._console_connection_status_element).text
 
     def get_remote_console_canvas(self):
-        return self._try_element_lookup(self._canvas_element).text
+        return self._try_element_lookup(self._canvas_element)
```

Comment:
The `.text` was added inadvertently about 7months ago. It is not required and causes tests to break. All 3 `get_ ` functions that are in this file need to return elements rather than `.text` for console tests to function correctly.